.env-production

```diff
@@ -0,0 +1,34 @@
+APP_ENV=production
+APP_KEY=base64:sMxf1KzJhttiy9Z9yORntUzNTzd1KoJlZRlgtfW6QeI=
+APP_DEBUG=false
+APP_LOG_LEVEL=debug
+APP_URL=http://darasini.herokuapp.com
```

Comment:
i think we shud stick to `darasani`

Reply:
Its taken on heroku. and i think you remember the .com domain is also taken.